cime_config/testdefs/testlist_clm.xml

```diff
@@ -4495,5 +4496,55 @@
     </options>
   </test>
 
+  <test name="SMS_Ld5" grid="f10_f10_mg37" compset="IHistClm60BgcCrop" testmods="clm/default">
+    <machines>
+      <machine name="derecho" compiler="intel" category="interim_restart"/>
+    </machines>
+    <options>
+      <option name="wallclock">00:20:00</option>
+    </options>
+  </test>
+
+  <test name="ERR_Ld7" grid="f10_f10_mg37" compset="IHistClm60BgcCrop" testmods="clm/default">
```

Comment:
This adds a total of 4 new tests, which in the spirit of keeping the aux_clm test list fast, might be good to do some shifting on. These tests should all be derecho as the production machine. But, it could be gnu rather than intel (even though intel is the default for production) which would save time since gnu tests are built parallel to the intel tests. And the intel list is longer than gnu. I also wonder if one or two of these should be shifted to the less frequent ctsm_sci testlist since that's about testing production features like this. Maybe the RTM test could move to that one. Maybe also move the IRT test to ctsm_sci, since it doesn't seem to be tested as a normal part of CESM alpha/beta testing. But, think about that and do what shifting you think makes sense.